proto/sync/sync.proto

```diff
@@ -2,28 +2,39 @@ syntax = "proto3";
 
 package sync;
 
+import "google/protobuf/empty.proto";
+import "rpcdb/rpcdb.proto";
+
 option go_package = "github.com/ava-labs/avalanchego/proto/pb/sync";
 
 // Request represents a request for information during syncing.
 message Request {
   oneof message {
-    RangeProofRequest range_proof_request = 1;
-    ChangeProofRequest change_proof_request = 2;
+    GetRangeProofRequest range_proof_request = 1;
+    GetChangeProofRequest change_proof_request = 2;
```

Comment:
I thought it'd be good to add `Get` to these to be consistent with the method names